packages/apify/src/internals/charging.ts

```diff
@@ -253,15 +265,17 @@ export class ChargingManager {
             throw new Error('ChargingManager is not initialized');
         }
 
-        if (!this.pricingInfo[eventName]) {
+        const price = this.isAtHome ? this.pricingInfo[eventName].price : 1; // Use a nonzero price for local development so that the maximum budget can be reached
```

Comment:
This feels weird and a step down from our reference implementation. We allowed passing in run ID so you can use real pricing for that project (I guess that would still work if we pass both `ACTOR_RUN_ID` and `IS_AT_HOME`), here we hardcode it at $1. I guess you would be ok to pass in the pricing somehow but we just didn't figure how?

Reply:
I really didn't like the idea that you first need to publish the actor to figure things out, we need to do better. What about having default pricing info in the configuration too. Would be used locally only, and we could figure out a way to set it up on platform if its not there (and override it with whatever is set up on the platform otherwise).

Reply:
It's a hard pass on copying the pricing from a platform run from me as well - it's just too involved and convoluted. But I can imagine that any alternative I'd implement would end up in endless bikeshedding, so I'd prefer to resolve this in a separate PR.

Reply:
Ok, lets resolve later, not a huge deal and there is workaround. Config file is good but Im afraid of support questions why it didnt propagate to platform.

Reply:
We could have an info log when we use the platform pricing and config has one. It would be right at the top, so quite easy to see.